src/components/modules/api/toolbar.ts

```diff
@@ -31,4 +32,36 @@ export default class ToolbarAPI extends Module {
   public close(): void {
     this.Editor.Toolbar.close();
   }
+
+  /**
+   * Toggles Block Setting of the current block
+   *
+   * @param {boolean} openingState —  opening state of Block Setting
+   */
+  public toggleBlockSettings(openingState?: boolean): void {
+    if (!this.Editor.BlockManager.currentBlockIndex) {
```

Comment:
```suggestion
    if (this.Editor.BlockManager.currentBlockIndex === -1) {
```

`BlockManager.currentBlockIndex` seems having `-1` or the index of current block.

Reply:
Done